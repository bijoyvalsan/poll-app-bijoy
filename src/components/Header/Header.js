import React from 'react';

import {StyledHeader} from './Header.styled';

const Header = () => (
  <StyledHeader>
    <h1>Poll App</h1>
  </StyledHeader>
);

export default Header;