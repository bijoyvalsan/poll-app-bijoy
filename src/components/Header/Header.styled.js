import styled from "styled-components";
import { device } from "../../theme";

export const StyledHeader = styled.header`
  background-color: black;
  color: white;
  padding: 24px 48px;
  h1 {
    margin: 0;
    padding: 0;
  }
  @media ${device.mobile} {
    padding: 12px 24px;
  }
`;
