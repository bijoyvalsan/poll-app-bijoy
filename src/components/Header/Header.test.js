import React from "react";
import { render } from "@testing-library/react";

import Header from "./Header";

describe("Header Component", () => {
  it("should render with default Header styles", () => {
    const { asFragment } = render(<Header>Poll App</Header>);
    expect(asFragment()).toMatchSnapshot();
  });
});
