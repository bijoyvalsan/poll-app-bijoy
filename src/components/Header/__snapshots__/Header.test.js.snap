// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render with default Header styles 1`] = `
<DocumentFragment>
  <header
    class="sc-bdnxRM giWmnp"
  >
    <h1>
      Poll App
    </h1>
  </header>
</DocumentFragment>
`;
